dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVDiagramPQ.mo

```diff
@@ -18,15 +18,24 @@ model GeneratorPVDiagramPQ "Model for generator PV based on SignalN for the freq
   extends BaseClasses.BaseQStator(QStatorPu(start = QGen0Pu * SystemBase.SnRef / QNomAlt));
 
 equation
-  when QGenPu <= QMinPu and UPu >= URefPu then
+  when QGenPu + QDeadBand  <= QMinPu and UPu - UDeadBand >= URefPu then
     qStatus = QStatus.AbsorptionMax;
     limUQDown = true;
     limUQUp = false;
-  elsewhen QGenPu >= QMaxPu and UPu <= URefPu then
+  elsewhen QGenPu - QDeadBand >= QMaxPu and UPu + UDeadBand <= URefPu then
     qStatus = QStatus.GenerationMax;
     limUQDown = false;
     limUQUp = true;
-  elsewhen (QGenPu > QMinPu or UPu < URefPu) and (QGenPu < QMaxPu or UPu > URefPu) then
+  // If the two following branches branch are not here we fail to adjust QGenPu if QMaxPu was modified but we were in Standard Mode.
```

Comment:
wrong comment?

Reply:
No, I added it to jutify adding branches to the when

Reply:
done